src/index.js

```diff
@@ -75,6 +76,21 @@ class CID {
     }
   }
 
+  get prefix () {
+    let v
+    if (this.version === 0) {
+      v = new Buffer('00', 'hex')
+    }
+    if (this.version === 1) {
+      v = new Buffer('01', 'hex')
+    }
+    return Buffer.concat([
+      v,
+      Buffer(codecVarints[this.codec]),
+      multihash.prefix(this.multihash)
+    ])
+  }
```

Comment:
This could be a bit more efficient. 

```js
return Buffer.concat([
  new Buffer(`0${this.version}`, 'hex'),
  codecVarints[this.codec],
  multihash.prefix(this.multihash)
])
```
as `codecVarints` should already be a Buffer.